applications/timer.go

```diff
@@ -3,7 +3,7 @@ package applications
 type LHTimer struct{}
 
 func (l LHTimer) ContainerName() string {
-	return "docker-compose_timer_1"
+	return "test_timer"
```

Comment:
is this what we want for production? Maybe we could patch this to output whatever we want so we can use it in both test and production?

Reply:
Cool, we'll think more about this.